tests/file.go

```diff
@@ -386,6 +387,24 @@ func wantedFromComment(p string) (directives []string, pkgPath, res, err, rops s
 	return
 }
 
+// readComments returns //-style comments from cg, but without truncating empty
+// lines like cg.Text().
+func readComments(cg *ast.CommentGroup) string {
+	var b strings.Builder
+	for _, c := range cg.List {
+		if len(c.Text) < 2 || c.Text[:2] != "//" {
+			// ignore no //-style comment
+			break
+		}
+		s := c.Text[2:]
+		if len(s) > 0 && s[0] == ' ' {
+			s = s[1:]
+		}
```

Comment:
```suggestion
		s := strings.TrimPrefix(c.Text[2:], " ")
```

I think it's more readable.